src/main/java/org/folio/event/service/UserOutboxService.java

```diff
@@ -233,7 +234,8 @@ private boolean isPersonalNotNull(Personal oldPersonal, Personal newPersonal) {
     return ObjectUtils.allNotNull(oldPersonal, newPersonal);
   }
 
-  private boolean isStaffUser(User user) {
-    return Objects.equals(STAFF.getTypeName(), user.getType());
+  private boolean isStaffOrSystemUser(User user) {
```

Comment:
what if user is null?

Reply:
When we work in consortia the 'type' field  is required and that validate before that validation.

Reply:
I am referring to the `User` object itself not the type property within.

Reply:
We use that validation after create/update user, and I think user can't be null here.